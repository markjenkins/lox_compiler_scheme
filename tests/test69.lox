// from craftinginterpreters/test/precedence.lox

// / has higher precedence than +.
print 2 + 6 / 3; // expect: 4
