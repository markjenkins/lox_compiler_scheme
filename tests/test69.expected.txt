OP_CONSTANT 2
OP_CONSTANT 6
OP_CONSTANT 3
OP_DIVIDE
OP_ADD
OP_PRINT
OP_RETURN
